feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/cross_chain/setup/CrossChainSetupFragment.kt

```diff
@@ -81,13 +81,6 @@ class CrossChainSetupFragment : BaseComposeBottomSheetDialogFragment<CrossChainS
             ).show(childFragmentManager)
         }
 
-        var constantDiff = 0
-        view.postDelayed({
-            val w = Rect()
-            view.getWindowVisibleDisplayFrame(w)
-            constantDiff = view.rootView.height - (w.bottom - w.top)
-        }, 100)
-
         view.viewTreeObserver.addOnGlobalLayoutListener {
```

Comment:
maybe completely remove it?

Reply:
soft keyboard visibility checks only left